core/pkg/artifacts/linker.go

```diff
@@ -72,3 +82,38 @@ func (al *ArtifactLinker) Link() error {
 
 	return err
 }
+
+func (al *ArtifactLinker) resolveOrgEntityName(portfolioEntity string, organization string) (string, error) {
+	orgFieldNames, err := gqlprobe.GetGraphQLFields(al.Ctx, al.GraphqlClient, "Organization")
+	if err != nil {
+		return "", err
+	}
+	canFetchOrgEntity := slices.Contains(orgFieldNames, "orgEntity")
+	switch {
+	case organization == "" && !canFetchOrgEntity:
+		return "", fmt.Errorf("Fetching Registry artifacts without inputting an organization is unavailable for your server version. Please upgrade your server to XX or later.")
```

Comment:
`XX` will be updated to the correct version number right?

Reply:
i guess we are going with version XX ;)

Reply:
was taking my time finding the version 0.50.0 added that in now!